index.bs

```diff
@@ -7652,6 +7652,122 @@ To <dfn abstract-op>Create a new supplemental public key record</dfn>, perform t
+             the {{CollectedClientData}} structure.
+     
+       1. pass-through the extension to the authenticator (see "client extension output" below)
+       1. pass-through the "authenticator extension output" to the caller as part of the assertion
+
+
+    <xmp class="idl">
+        dictionary CollectedClientConfirmationData : CollectedClientData {
+            required CollectedClientAdditionalConfirmationData confirmationPrompt;
+        };
+    </xmp>
+
+    The {{CollectedClientConfirmationData}} dictionary inherits from
+    {{CollectedClientData}}. It contains the following additional field:
+
+    <dl dfn-type="dict-member" dfn-for="CollectedClientConfirmationData">
+        :  <dfn>confirmationPrompt</dfn> member
+        :: The full context of the confirmation to give.
+    </dl>
+
+
+    <xmp class="idl">
+        dictionary CollectedClientAdditionalConfirmationData {
+            required USVString confirmationPrompt;
+        };
+    </xmp>
+
+    The {{CollectedClientAdditionalConfirmationData}} dictionary contains the following
+    fields:
+
+    <dl dfn-type="dict-member" dfn-for="CollectedClientAdditionalConfirmationData">
+        :  <dfn>confirmationPrompt</dfn> member
+        :: The full context of the confirmation to give.
+    </dl>
+
+: Client extension output
+:: Returns the authenticator extension output string UTF-8 decoded into a USVString.
+    <xmp class="idl">
+    partial dictionary AuthenticationExtensionsClientOutputs {
+      USVString confirmationPrompt;
+    };
+    </xmp>
+
+: Authenticator extension input
+:: The client provides the extension input encoded as a CBOR text string (major type 3).
+
+<pre>
+CDDL:
+confirmationPromptInput = (tstr)
+</pre>
+
+: Authenticator extension processing
+:: The authenticator supporting this extension MUST display the confirmation prompt to the user
+    before performing either [=user verification=] or [=test of user
+    presence=]. The authenticator MAY wrap lines that are too wide to be shown if needed.
+
+: Authenticator extension output
+:: If the authenticator supports this extension (and only then): A single CBOR string, representing the
+    confirmation prompt.
+
+<pre>
+CDDL:
+confirmationPromptOutput = (tstr)
+</pre>
+
+#### `confirmation` Extension Output Verification Procedures #### {#sctn-confirmation-extension-verification}
+
+Verifying the <code>[=confirmation=]</code> extension output is performed by the [=[RP]=] whenever a the use of <code>[=confirmatione=]</code> was requested.
+
+##### Authentication (`get()`) ##### {#sctn-confirmation-extension-verification-get}
+
+This extension is only specified to be used in conjunction with a {{CredentialsContainer/get()|navigator.credentials.get()}} call.
+
+If the [=[RP]=] requested the `confirmation` extension in a {{CredentialsContainer/get()|navigator.credentials.get()}} call, then the below verification step are performed in the context of [step 19](#authn-ceremony-verify-extension-outputs)
```

Comment:
While used in the `supplementalPubKeys` extension, this section structure is not (yet?) a general pattern, so I think we can just collapse these to one. We also shouldn't need to restate here that it's only relevant for assertions.

```suggestion
#### `confirmation` Extension Output Verification Procedure #### {#sctn-confirmation-extension-verification}

The below verification step are performed in the context of [step 19](#authn-ceremony-verify-extension-outputs)
```